src/main/java/ru/skypro/flea/entity/Comment.java

```diff
@@ -0,0 +1,39 @@
+package ru.skypro.flea.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import ru.skypro.flea.converter.LocalDateTimeConverter;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "comments")
+@NoArgsConstructor
+@Getter
+@Setter
+public class Comment {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "id", nullable = false)
+  private Integer id;
+
+  @Column(name = "text", nullable = false)
+  private String title;
+
+  @Column(name = "pubic_date")
+  @Convert(converter = LocalDateTimeConverter.class)
+  @NotNull
+  private LocalDateTime pubicDate;
```

Comment:
to set creation time automatically we can write such a method:
```java
@PrePersist
private void onCreate() {
  publicDate = LocalDateTime.now();
}
```
it should work like this: method runs in the very last moment before persisting object to DB. method sets the **publicDate** according to the time of specified moment.